cluster/gce/util.sh

```diff
@@ -912,6 +912,19 @@ function detect-subnetworks() {
   echo "${color_red}Could not find subnetwork with region ${REGION}, network ${NETWORK}, and project ${NETWORK_PROJECT}"
 }
 
+function delete-all-firewall-rules-in-network() {
+  if fws=$(gcloud compute firewall-rules list --project "${NETWORK_PROJECT}" --filter="network=${NETWORK}" --format="value(name)"); then
+    if [[ -z "${fws}" ]]; then
+      echo "No firewall rules left to be deleted."
+    else
+      echo "Deleting firewall rules remaining in network ${NETWORK}: ${fws}"
+      delete-firewall-rules "$fws"
```

Comment:
Are you sure this works as intended? i.e "$fws" is passed as a list of arguments to `delete-firewall-rules` instead of a single space-separated string?

Reply:
Yep, it worked on my test script, let me rerun it to confirm.